zookeeper-command-line-client/src/main/sh/vespa-zkflw

```diff
@@ -77,6 +77,24 @@ export ROOT
 
 # END environment bootstrap section
 
+# Defaults
+server="localhost"
+port="2181"
+command="stat"
+
+if [ "$#" -eq 1 ]; then
+    command=$1
+elif [ "$#" -eq 2 ] || [ "$#" -gt 3 ]; then
+    echo "Need 0, 1 (command) or 3 arguments (host, port, command)"
+    exit 1
+elif [ "$#" -eq 3 ]; then
+    server=$1
+    port=$2
+    command=$3
+fi
+
+args="$server $port $command"
+
 java -cp ${VESPA_HOME}/lib/jars/zookeeper-command-line-client-jar-with-dependencies.jar \
      -Dlogback.configurationFile="logback-vespa.xml" -Xms32m -Xmx512m \
-     com.yahoo.vespa.zookeeper.cli.FourLetterWordMain "$@"
+     com.yahoo.vespa.zookeeper.cli.FourLetterWordMain $args
```

Comment:
If command ever contains something that means something special in shell, e.g. ;, \, quotes, etc, then this will not work.  The correct way of preserving arguments would be to make args an array, and then do e.g. args=("$1" "$2" "$3").

Reply:
Thanks, that's better, fixd